lib/movie/screens/movie_details_screen/movie_details_screen.dart

```diff
@@ -0,0 +1,315 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:get_it/get_it.dart';
+
+import 'package:movie/config.dart';
+import 'package:movie/infrastructure/theme/theme_extensions.dart';
+import 'package:movie/movie/domain/movie.dart';
+import 'package:movie/movie/screens/movie_details_screen/cubit/movie_details_cubit.dart';
+import 'package:movie/utils/date_time_formatting_extension.dart';
+
+// ---Texts---
+const _kScore = 'Score:';
+const _kRating = 'Rating:';
+const _kReleaseDate = 'Release Date:';
+
+// ---Parameters---
+const _kPadding = 15.0;
+const _kHeight = 30.0;
+const _kHeightLine = 1.0;
+const _kWidth = 20.0;
+const _kFontSize = 22.0;
+const _kFontSizeText = 18.0;
+const _kHeightContainer = 280.0;
+const _kWidthContainer = 200.0;
+const _kRadius = 20.0;
+
+class MovieDetailsScreen extends StatefulWidget {
+  static const _routeName = '/movie-details-screen';
+
+  static PageRoute<MovieDetailsScreen> route(int id, String title) {
+    return MaterialPageRoute(
+      settings: const RouteSettings(name: _routeName),
+      builder: (context) {
+        return BlocProvider(
+          create: (context) => MovieDetailsCubit(MovieDetailsInitial()),
+          child: MovieDetailsScreen(id: id, title: title),
+        );
+      },
+    );
+  }
+
+  final int id;
+  final String title;
+  const MovieDetailsScreen({
+    Key? key,
+    required this.id,
+    required this.title,
+  }) : super(key: key);
+
+  @override
+  State<MovieDetailsScreen> createState() => _MovieDetailsScreenState();
+}
+
+class _MovieDetailsScreenState extends State<MovieDetailsScreen> {
+  MovieDetailsCubit get movieCubit =>
+      BlocProvider.of<MovieDetailsCubit>(context);
+
+  @override
+  void initState() {
+    movieCubit.getMovie(widget.id);
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
wrap whole `Scaffold` into `BlocBuilder` widget and remove `BlocBuilder` from `body`